src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/csip/CSIPDpsApiService.kt

```diff
@@ -72,6 +72,14 @@ class CSIPDpsApiService(@Qualifier("csipApiWebClient") private val webClient: We
       .awaitBodilessEntity()
   }
 
+  suspend fun deleteCSIPAttendee(csipAttendeeId: String) {
```

Comment:
After further investigation - this change may not be needed - however I have completed the PR incase we do need it and also there is some name refactoring